### ISSUE TITLE
Links not working

### ISSUE THREAD

**Issue by sigdestad** [enonic/app-siteimprove#21]
When on site level, clicking the panel links to visit siteimprove gives a 404 page
* QA panel
* Accessibility panel
* SEO panel

**Comment by alansemenov** [comment]
Looks like Siteimprove made some changes that broke this.
The way it worked before is as follows:
1. Each site under Siteimprove account has siteId, returned by API request.
2. The general dashboard opens via ``https://my2.siteimprove.com/Dashboard/<SiteId>/Dci/Index`` - this still works
3. Dashboards of the three sections (QA, Accessibility and SEO) used to open via similar links, for example for QA: ``https://my2.siteimprove.com/QualityAssurance/<SiteId>/Dci/Index``. 

Now this stopped working. If you try to open pages for QA, Accessibility and SEO manually in the Siteimprove admin panel, you will see that ``siteId`` changes in the url to a totally different value which we cannot retrieve via API. 

For example, for Enonic:
https://my2.siteimprove.com/Dashboard/7730629732/Dci/Index (general dashboard, 7730629732 is ``siteId`` that comes from API)
https://my2.siteimprove.com/QualityAssurance/794903/Overview/Index (here **794903** is a totally different number which is nowhere to be found)

I will temporary hide icons opening external links to these three sections but we have to ask Siteimprove to either fix it back to how it was before, or explain what this second number is and how we can get hold of it via API.

**Comment by sigdestad** [comment]
ok for me!

**Comment by alansemenov** [comment]
@edloidas For the site we need to retrieve links to QA, Accessibility and SEO in the same way we do it for pages - first we need to retrieve ``pageId`` for the root page, then fetch page summary for the root page and get links from the response.

![image](https://user-images.githubusercontent.com/11612490/50347684-ba17b700-0536-11e9-820c-638021dd09c2.png)
